testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -226,14 +223,16 @@ def test_AlignTraj_outfile_default_exists(self, universe, reference, tmpdir):
         fitted = mda.Universe(PSF, outfile)
 
         # ensure default file exists
-        with mda.Writer(str(tmpdir.join('rmsfit_align_test.dcd')),
```

Comment:
not calling `str()` is messing up the py2.7 tests

Reply:
Wooops, that was a leftover for the first attempt. I reverted it to the original syntax.